operations/mimir-mixin/docs/playbooks.md

```diff
@@ -214,6 +214,23 @@ How to **investigate**:
         - If memcached eviction rate is high, then you should scale up memcached replicas. Check the recommendations by `Mimir / Scaling` dashboard and make reasonable adjustments as necessary.
         - If memcached eviction rate is zero or very low, then it may be caused by "first time" queries
 
+#### Alertmanager
+
+How to **investigate**:
+
+- Check the `Mimir / Alertmanager` dashboard
+  - Looking at the dashboard you should see which part of the stack is affected
+- Deduce where in the stack the latency is being introduced
+  - **`Configuration API (gateway) + Alertmanager UI`**
+    - Latency may be caused by the time taken for the gateway to receive the entire request from the client. There are a multitude of reasons this can occur, so communication with the user may be necessary. For example:
+      - Network issues such as packet loss between the client and gateway.
+      - Poor performance of intermediate network hops such as load balancers or HTTP proxies.
+      - Client process having insufficient CPU resources.
+    - The gateway may need to be scaled up. Use the `Mimir / Scaling` dashboard to check for CPU usage vs requests.
+    - There could be a problem with authentication (eg. slow to run auth layer)
+  - **`Distributor`**
```

Comment:
They're not the name of the actual components, so removing the code style. Also clarify it's the alertmanager distributor.

```suggestion
  - **Alertmanager distributor**
```